### PR TITLE
actions: add renovate

### PR DESCRIPTION
**Pull request by jhaals**
replaces dependabot with renovate for dependency updates

### CODE REVIEW

File: .github/workflows/sync_renovate-changesets.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                  "show",
+                  file,
+                ]);
+                for (const change of changes.split("\n")) {
+                  if (!change.startsWith("+ ")) {
+                    continue;
+                  }
+                  const match = change.match(/"(.*?)"/g);
+                  bumps.set(match[0].replace(/"/g, ""), match[1].replace(/"/g, ""));
+                }
```
Comment (Rugvip) [inline .github/workflows/sync_renovate-changesets.yml:64]: 🤌

**Review by Rugvip — APPROVED** [review]
Perhaps we should keep both of them for now until we've moved over?

File: .github/workflows/sync_renovate-changesets.yml (hunk 1 above)
Comment (jhaals) [reply .github/workflows/sync_renovate-changesets.yml:64]: My finest work 👨‍🍳 :trollface:

**Comment by jhaals** [comment]
kept dependabot. Added renovate config. I did not add any package rules yet

**Pull request merged**